src/app/components/CallToActionLinkWithChevron/index.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { PropsWithChildren } from 'react';
+
+import { SerializedStyles } from '@emotion/react';
+import CallToActionLink from '.';
+import { CallToActionLinkProps } from '../CallToActionLink/types';
+
+const Component = ({
+  href,
+  children,
+}: PropsWithChildren<
+  CallToActionLinkProps & { chevronStyles?: () => SerializedStyles }
+>) => <CallToActionLink href={href}>{children}</CallToActionLink>;
+
+export default {
+  title: 'Components/Call To Action Link With Chevron',
+  Component,
+};
+
+export const Example = () => {
+  return <Component href="www.bbc.com/afrique">Call To Action</Component>;
```

Comment:
I try to use this generic link in all stories
```suggestion
  return <Component href="https://www.bbc.com/ws/languages">Call To Action</Component>;
```